rbac/management/migrations/0011_group_naming.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.4 on 2019-11-26 17:03
+
+from django.db import migrations, models
+
+
+def update_default_name(apps, schema_editor):
+    Group = apps.get_model('management', 'Group')
+    for group in Group.objects.all():
+        if group.system:
+            group.name = "Default user access"
+        else:
+            group.name = "Custom default user access"
```

Comment:
I don't think we want to update all groups, do we? We just need to ensure that currently, any 'system' group is "Default user access". Then on update, we'll bump it to "Custom default user access", and flip the `system` flag but the data migration just needs to effectively sync the data as it is now. 

Otherwise normal groups will pickup the name "Custom default user access", and actually I would think that might break since there's a unique constraint on group names?

Reply:
Oh, yeah. I was thinking about current, formerly system groups, but this is a bit broad for that and those may not really be a concern. I'll just strike the else.